src/api-reference/authentication/apidoc.markdown

```diff
@@ -109,7 +109,7 @@ Name | Type | Format | Description
 `client_id`|`string` | `UUID` | **Required** The client applications client_id supplied by App Management
 `client_secret`|`string` | `UUID` | **Required** The client applications client_secret supplied by App Management
 `refresh_token`|`string` | `UUID` | **Required** An existing valid refresh token to be used to request a new access token
-`scope`|`string` | | The client applications list of scopes
+`scope`|`string` | | **Optional** The client applications list of scopes
```

Comment:
```suggestion
`scope`|`string` |-| **Optional** The client applications list of scopes
```
We should denote an intentional blank cell with use of the `-`.

I would prefer we clear up the whitespace in the table as well.